content/quiz_events.md

```diff
@@ -113,19 +114,20 @@ Individual Fouls
 : A foul awarded to one or more individual quizzers.
 
 Team Fouls
-: The sum of individual fouls.
+: The sum of individual fouls by members of the same team.
 
 ## Time-Outs
 
     Set the timeout duration to 60.
 
-- Each team is allowed two, one-minute time-outs.  Only one of those time-outs may be taken during questions 18-20. 
+- Each team is allowed two, one-minute time-outs.  Only one of those time-outs may be taken after the numbered question 17 (including any 17A or 17B).
```

Comment:
Is it worth defining questions like this somewhere to help reduce explanation elsewhere in the rulebook

Question = Numbered Question & A/B Questions
Numbered Question = A question labeled by only a number
A/B Question: A question labeled by both a number and a letter being either "A" or "B"

Reply:
The wording for just plain question definitely need tweaking. I'm struggling to come up with a word choice outside of question block, question number. 

Question Number vs Numbered Question could also be used.

Reply:
How do we differentiate the definition of "Question" in the scoring context from the definition of "Question" in the "Question/Answer/Question Text/Answer Text" context?

Reply:
Maybe something like Question block does make sense. It can be defined by everything below it.

- A Question block is the sum of the elements listed below
     - Numbered Question
     - Question
          - Question Text
     - Answer
          - Answer Text
     - A/B Question
          - Applicable on questions beginning at 16

Reply:
Or does that actually just dirty the language up.

Reply:
I'd have the think about it. I think Scoring and "the quiz question" are very different contexts, which is why it sucks that "Question" is the PERFECT word for concepts within each.

Reply:
I believe we need to do 3 things:

- Use the same terms and associated definitions universally
- Avoid any overloading of terms
- Reuse terms and associated definitions generally held by Quizzingdom (i.e. "question" isn't just the question)

Therefore:

- I think "question" needs to refer to the whole (question text, answer text, reference, type, and "number").
- Then "number" needs to contain all possibilities, be they "numeric" (i.e. 19) or "alphanumeric" (i.e. 19A).

We can add additional sub-terms as needed. So for example, you could say that regarding question 19A, the "A" is the alpha component of the question number, or just "question number alpha" for short.

Reply:
I'm going to tackle these definitions and re-word in a fresh PR.